crates/sui-benchmark/tests/simtest.rs

```diff
@@ -234,8 +235,10 @@ mod test {
         );
 
         let bank = BenchmarkBank::new(proxy.clone(), primary_gas, pay_coin);
+        let protocol_config = sui_protocol_config::ProtocolConfig::get_for_max_version();
```

Comment:
@mystenmark this line seems to be causing failures in CI. Any ideas how we can get around it?